java/core/src/test/java/sleeper/core/statestore/FileReferenceFactory.java

```diff
@@ -67,18 +114,46 @@ public static FileReferenceFactory fromUpdatedAt(StateStore stateStore, Instant
         }
     }
 
+    /**
+     * Creates a file in the root partition.
+     *
+     * @param  records the number of records in the file
+     * @return         the file reference
+     */
     public FileReference rootFile(long records) {
         return fileForPartition(partitionTree.getRootPartition(), records);
```

Comment:
We could add a comment after the summary here saying that the partition ID will be used as the filename, so if you make multiple calls to this method it will fail to add subsequent files since they all have the same filename.

Reply:
Done